File: APEC.RRHH.api/Core/Models/CandidateEmployee.cs
using System;
using System.Collections.Generic;
using Core.Contracts;
using Core.Enums;

namespace Core.Models
{
    public sealed class CandidateEmployee : IEntityBase
    {
        public Guid Id { get; set; }
        public User User { get; set; }
        public Guid UserId { get; set; }
        public FeatureStatus Status { get; set; }
        public ISet<CandidateInterview> Interviews { get; set; }
        public ISet<CandidateEmployeeAspiratedJob> CandidateEmployeeAspiratedJobs { get; set; }
    }
}

File: APEC.RRHH.webapp/src/app/models/candidate-employee.ts
import { FeatureStatus } from '../enums/feature-status';
import { CandidateEmployeeAspiratedJob } from './candidate-employee-aspirated-job';
import { CandidateInterview } from './candidate-interview';
import { UserProfile } from './user-profile';

export class CandidateEmployee {
    id: string;
    userId: string;
    user: UserProfile;
    status: FeatureStatus;
    interviews: CandidateInterview[];
    candidateEmployeeAspiratedJobs: CandidateEmployeeAspiratedJob[];
}
